projects/core/src/user/connectors/order/user-order.connector.ts

```diff
@@ -21,4 +26,8 @@ export class UserOrderConnector {
   ): Observable<OrderHistoryList> {
     return this.adapter.loadHistory(userId, pageSize, currentPage, sort);
   }
+
+  protected isGuid(str: string): boolean {
```

Comment:
Why is this protected?

Reply:
we don't want it to be extensible?

Reply:
I think it should be only if there is a possibility that the pattern for GUID can be different.

Reply:
There is another way to do it:
change `loadOrderDetails(orderCode: string)` to ``loadOrderDetails(userId: string, orderCode: string)`. So, we read user token first, if user not login, then userId is 'anonymous'. In this way, no need check ID "isGuid".

Reply:
GUID pattern is set in `spring.xml`. Technically, it can be different.